src/lib/modules/solidity/index.js

```diff
@@ -143,6 +143,7 @@ class Solidity {
             compiled_object[className].gasEstimates = contract.evm.gasEstimates;
             compiled_object[className].functionHashes = contract.evm.methodIdentifiers;
             compiled_object[className].abiDefinition = contract.abi;
+            compiled_object[className].userdoc = contract.userdoc;
```

Comment:
Do we also have to change the solc plugin?

Reply:
No. This is simply to support the sender override introduced in `remix_tests`.

Reply:
Re: the above - this is where the override sender was introduced: https://github.com/ethereum/remix/blob/a04880946ab5735159c91b7df126fba9c929c62b/remix-tests/src/testRunner.js#L14-L17